### ISSUE TITLE
ssd_mobilenet_v1_quantized_300x300_coco14_sync's mAP=0

### ISSUE THREAD

**Issue by cococyx** [tensorflow/tensorflow#22336]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform Ubuntu 16.04
- **TensorFlow installed from source: models's commit = 55d55abc71483723743c0273b9c1fd8e0c7d8391 and tensorflow's commit = 6c3e1f4bc803f2dc8a804f4f15ada0eda6c90a18
- **TensorFlow version **:1.9.0
- **Python version**:Python 3.5.2 (default, Nov 23 2017, 16:37:01)
- **Bazel version (if compiling from source)**: 0.11.1
- **GCC/Compiler version (if compiling from source)**: gcc (Ubuntu 5.4.0-6ubuntu1~16.04.10) 5.4.0 20160609
- **CUDA/cuDNN version**: Cuda compilation tools, release 9.0, V9.0.176;cuDNN7.0
- **GPU model and memory**: GeForce GTX 1070 Ti  * 4
- **Exact command to reproduce**:
python object_detection/model_main.py --pipeline_config_path=./object_detection/samples/configs/ssd_mobilenet_v1_quantized_300x300_coco14_sync.config  --model_dir=/checkpoint/ssd_mobilenet_v1_quantized_300x300_coco14_sync --checkpoint_dir=/ssd/ssd_mobilenet_v1_quantized_300x300_coco14_sync_2018_07_18 --run_once=True

### Describe the problem
0.
train		118287
wget -c http://images.cocodataset.org/zips/train2017.zip 
wget -c http://images.cocodataset.org/annotations/annotations_trainval2017.zip
eval		5000
wget -c http://images.cocodataset.org/zips/val2017.zip 
wget -c http://images.cocodataset.org/annotations/stuff_annotations_trainval2017.zip
1
Download checkpoint in http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_quantized_300x300_coco14_sync_2018_07_18.tar.gz
2
Amend file's path in models/object_detection/samples/configs/ssd_mobilenet_v1_quantized_300x300_coco14_sync.config
3.
from models/research,
python object_detection/model_main.py --pipeline_config_path=./object_detection/samples/configs/ssd_mobilenet_v1_quantized_300x300_coco14_sync.config  --model_dir=/checkpoint/ssd_mobilenet_v1_quantized_300x300_coco14_sync --checkpoint_dir=/ssd/ssd_mobilenet_v1_quantized_300x300_coco14_sync_2018_07_18 --run_once=True

### Source code / logs
1
[ssd_mobilenet_v1_quantized_300x300_coco14_sync.zip](https://github.com/tensorflow/tensorflow/files/2391202/ssd_mobilenet_v1_quantized_300x300_coco14_sync.zip)
2
[ssd_mobilenet_v1_quantized_300x300_coco14_sync_log.txt](https://github.com/tensorflow/tensorflow/files/2391211/ssd_mobilenet_v1_quantized_300x300_coco14_sync_log.txt)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
OS Platform and Distribution
Mobile device

**Comment by harshini-gadige** [comment]
@cococyx Hi, Could you please explain the issue in brief ? Also appreciate if you provide a code snippet to reproduce the issue.

**Comment by cococyx** [comment]
OS Platform and Distribution :Ubuntu 16.04 , not Mobile device

**Comment by cococyx** [comment]
And ssd_mobilenet_v1_quantized_300x300_coco14_sync.config is attached ssd_mobilenet_v1_quantized_300x300_coco14_sync.zip

diff --git a/models/research/object_detection/samples/configs/ssd_mobilenet_v1_quantized_300x300_coco14_sync.config b/models/research/object_detection/samples/configs/ssd_mobilenet_v1_qua                                                  ntized_300x300_coco14_sync.config
index a79d772..ed054ab 100644
--- a/models/research/object_detection/samples/configs/ssd_mobilenet_v1_quantized_300x300_coco14_sync.config
+++ b/models/research/object_detection/samples/configs/ssd_mobilenet_v1_quantized_300x300_coco14_sync.config
@@ -139,7 +139,8 @@ model {
 }

 train_config: {
-  fine_tune_checkpoint: "PATH_TO_BE_CONFIGURED/model.ckpt"
+  #fine_tune_checkpoint: "/data/checkpoint/ssd/ssd_mobilenet_v1_quantized_300x300_coco14_sync_2018_07_18/model.ckpt"
+  fine_tune_checkpoint: "/data/checkpoint/ssd/ssd_mobilenet_v1_coco_2018_01_28/model.ckpt"
   batch_size: 128
   sync_replicas: true
   startup_delay_steps: 0
@@ -172,29 +173,29 @@ train_config: {
 }
 train_input_reader: {
   tf_record_input_reader {
-    input_path: "PATH_TO_BE_CONFIGURED/mscoco_train.record-00000-of-00100"
+    input_path: "/data/dataset/coco/output_train/coco_train.record-?????-of-00100"
   }
-  label_map_path: "PATH_TO_BE_CONFIGURED/mscoco_label_map.pbtxt"
+  label_map_path: "/home/DLA_TF19_Train/models/research/object_detection/data/mscoco_label_map.pbtxt"
 }

 eval_config: {
   metrics_set: "coco_detection_metrics"
   use_moving_averages: false
-  num_examples: 8000
+  num_examples: 5000
 }

 eval_input_reader: {
   tf_record_input_reader {
-    input_path: "PATH_TO_BE_CONFIGURED/mscoco_val.record-00000-of-00010"
+    input_path: "/data/dataset/coco/output_val/coco_val.record-?????-of-00010"
   }
-  label_map_path: "PATH_TO_BE_CONFIGURED/mscoco_label_map.pbtxt"
+  label_map_path: "/home/DLA_TF19_Train/models/research/object_detection/data/mscoco_label_map.pbtxt"
   shuffle: false
   num_readers: 1
 }

 graph_rewriter {
   quantization {
-    delay: 48000
+    delay: 4800
     activation_bits: 8
     weight_bits: 8
   }

**Comment by cococyx** [comment]
This is easy to reproduce by

python object_detection/model_main.py --pipeline_config_path=./object_detection/samples/configs/ssd_mobilenet_v1_quantized_300x300_coco14_sync.config --model_dir=/checkpoint/ssd_mobilenet_v1_quantized_300x300_coco14_sync --checkpoint_dir=/ssd/ssd_mobilenet_v1_quantized_300x300_coco14_sync_2018_07_18 --run_once=True
 
from models/research

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 61 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by NorwayLobster** [comment]
hi bro @cococyx , do you solve the problem. i got a similar problem. the all of AP/AR are almost 0 when I train ssd using .config: ssd_mobilenet_v2_quantized_300x300_coco_2018_09_14.config and its corresponding pre-trained checkpoint.
could someone help or give some hint? thank you

**Comment by petewarden** [comment]
This bug is a couple of years old and the assignee is no longer working in this area, so closing this since we're unlikely to get to it. Please reopen if you are still hitting this problem.
